docs/can-guides/topics/data/managing-sessions/managing-sessions.md

```diff
@@ -16,6 +16,8 @@ This guide will explain how:
 - to use [can-connect/can/session/session can/session] with application held tokens 
 - to initialize [can-connect/can/session/session can/session] manually if you know you have an active session
 
+If you prefer a video introduction to this material, the basics of this guide are also covered in a presentation from our monthly online meetup, [available here](https://www.youtube.com/watch?v=sIh0_hCx6WM).
```

Comment:
We embed videos in a [couple of](https://canjs.com/doc/guides/recipes/credit-card-advanced.html) [our guides](https://canjs.com/doc/guides/recipes/playlist-editor.html). Unless it adds too much weight to the page, I would consider embedding the video here with something like:

```html
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/sIh0_hCx6WM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```